jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ServletContextResponse.java

```diff
@@ -89,6 +90,12 @@ public ServletContextResponse(ServletChannel servletChannel, ServletContextReque
         _httpServletResponse = newServletApiResponse();
     }
 
+    @Override
+    public Request getRequest()
+    {
+        return _request;
+    }
```

Comment:
```suggestion
    @Override
    public ServletContextRequest getRequest()
    {
        return (ServletContextRequest)super.getRequest();
    }
```

Also both the `_request` and `_response` fields are duplicates of what exists already in base classes, so there is no need to duplicate them (saves runtime memory).